examples/default/flake.nix

```diff
@@ -1,36 +1,59 @@
 {
-  description = "Fleek Configuration";
-
   inputs = {
-    # Nixpkgs
+    # Principle inputs (updated by `nix run .#update`)
     nixpkgs.url = "github:nixos/nixpkgs/nixos-unstable";
-
-    # Home manager
     home-manager.url = "github:nix-community/home-manager";
     home-manager.inputs.nixpkgs.follows = "nixpkgs";
+
+    flake-parts.url = "github:hercules-ci/flake-parts";
+    nixos-flake.url = "github:bketelsen/nixos-flake";
+    fleek.url = "github:ublue-os/fleek";
   };
 
-  outputs = { nixpkgs, home-manager, ... }@inputs: {
+  outputs = inputs@{ self, ... }:
+    let
+      inherit (self) outputs;
+    in
+    inputs.flake-parts.lib.mkFlake { inherit inputs; } {
+      systems = [ "x86_64-linux" "aarch64-linux" "aarch64-darwin" "x86_64-darwin" ];
```

Comment:
This an be made shorter using [nix-systems](https://github.com/nix-systems/nix-systems)

Eg: https://github.com/srid/haskell-template/blob/bdd135025466fc82fbc88a46043c53ec28930979/flake.nix#L16